### PR TITLE
PHP 8.3 support added

### PR DESCRIPTION
**Pull request by glo71317**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### CODE REVIEW

**Review by gsteel — CHANGES_REQUESTED** [review]
In order to run tests on 8.3, you'll need to ignore platform requirements for 8.3 in `.laminas-ci.json` too

**Comment by glo71317** [comment]
> In order to run tests on 8.3, you'll need to ignore platform requirements for 8.3 in `.laminas-ci.json` too

I have updated

File: .laminas-ci.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "ignore_php_platform_requirements": {
         "8.0": false,
         "8.1": false,
-        "8.2": false
+        "8.2": false,
+        "8.3": false
```
Comment (gsteel) [inline .laminas-ci.json:11]: This needs to be `true` in order to actually ignore platform requirements:

https://github.com/laminas/laminas-ci-matrix-action?tab=readme-ov-file#configuration

**Review by gsteel — CHANGES_REQUESTED** [review]

**Comment by gsteel** [comment]
Thanks @glo71317 but this has been handled in #226

**Pull request closed without merging**